packages/lisk-p2p/test/integration/p2p.ts

```diff
@@ -58,6 +59,18 @@ describe('Integration tests for P2P library', () => {
 				expect(p2p).to.have.property('isActive', true);
 			});
 		});
+
+		describe('P2P.request', () => {
+			it('should throw an error when not able to get any peer in peer selection', async () => {
```

Comment:
does user have control over if peer can be obtained in selection or not?
or was if it at least has seed, this one happen?

Reply:
No, user doesn't have control over the default peer selection but I think that needs to be covered as well where the user is passing its own peer selection function. But here, we are just testing in a completely disconnected network with no peers and seed nodes, we expect it to throw this error.